src/Console/Application.php

```diff
@@ -109,22 +110,50 @@ public function doRun(InputInterface $input, OutputInterface $output): int
         return $result;
     }
 
-    public function getLongVersion(): string
+    /**
+     * @internal
+     */
+    public static function getAppAbout(bool $decorated = false): string
```

Comment:
```suggestion
    public static function getAbout(bool $decorated = false): string
```

for the same reasons - `Application::getAppAbout()` looks weird and there's `getLongVersion()`, not `getAppLongVersion()`.

Personally I would use just `Application::about()`, but I know your preference about getters 😝.